hist/hist/src/TGraph.cxx

```diff
@@ -2146,26 +2146,17 @@ void TGraph::SaveAs(const char *filename, Option_t *option) const
 
 void TGraph::SavePrimitive(std::ostream &out, Option_t *option /*= ""*/)
 {
-   out << "   " << std::endl;
    static Int_t frameNumber = 0;
    frameNumber++;
 
-   TString fXName, fYName;
-
+   TString args;
    if (fNpoints >= 1) {
-      fXName = SaveArray(out, "fx", frameNumber, fX);
-      fYName = SaveArray(out, "fy", frameNumber, fY);
+      auto xname = SaveArray(out, "fx", frameNumber, fX);
+      auto yname = SaveArray(out, "fy", frameNumber, fY);
```

Comment:
```suggestion
      TString xname = SaveArray(out, "fx", frameNumber, fX);
      TString yname = SaveArray(out, "fy", frameNumber, fY);
```
The next line require `xname` and `yname` to be `TString`, it is probably better to fail compilation here if `SaveArray` changes return type.